### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by terrajobst**
> Someone once told me that time was a predator that stalked us all our lives. But I rather believe that time is a companion who goes with us on the journey and reminds us to cherish every moment because they'll never come again. What we leave behind is not as important as how we've lived. After all, we're only mortal.
> 
> -- Jean-Luc Picard

### CODE REVIEW

**Comment by benaadams** [comment]
It has code now; marvelous types like [TwoGBFileStream](https://github.com/benaadams/aspnetcore.ben/blob/master/samples/OverlappedFileStream/TwoGBFileStream.cs)

**Comment by terrajobst** [comment]
You changed the purpose and strategy of this repo?!?! WHERE IS THE RFC THAT INVOLVED THE COMMUNITY? HOW DARE YOU MAKING DECISIONS ON YOUR OWN. WHO DO YOU THINK YOUR ARE? THE OWNER OF THIS REPO?!

**Comment by benaadams** [comment]
Its a really fast file though...

File: README.md [hunk 1]
```diff
@@ -1,2 +1,18 @@
...
+        Once upon a time I had a dream. A dream of the perfect framework that helps you get your job done
+        better, faster, more realiably.
+
+        During my quest, I realized that my way of thinking is wrong. It's not about the framework. It's
+        not about how sexy your code reads. It's about finding your way. And the best way to find your
+        way is by writing your own framework. Use this template to start your effort. Once you're done,
+        your framework should become the same as this template: empty.
+
+        This comment shall remain for the truly interested. The people willing to look behind the curtain,
+        the poeple willing to dig deep, the people willing to delete code to reach the nirvana of programming.
```
Comment (cH40z-Lord) [inline README.md]: `poeple` = `people`

**Comment by terrajobst** [comment]
Why is this not being merged?

**Comment by benaadams** [comment]
Two close commits; seemed a bit volatile, was waiting for it to stabilize

**Pull request merged**